tests/tests/test_mtls.py

```diff
@@ -76,7 +76,59 @@ def make_script_artifact(artifact_name, device_type, output_path):
 
 
 class TestClientMTLSEnterprise:
-    def common_test_mtls_enterprise(self, setup_ent_mtls, algorithm=None):
+    def get_key_uri_hsm(self, ssl_engine_id, device):
+        algorithm = "rsa"
+        tmpdir = tempfile.mkdtemp()
+        key = f"/var/lib/mender/client.1.{algorithm}.key"
+        pin = "0001"
+        script = f"""\
+#!/bin/bash
+echo "module: /usr/lib/softhsm/libsofthsm2.so" > /usr/share/p11-kit/modules/softhsm2.module
+mkdir -p /softhsm/tokens
+echo "directories.tokendir = /softhsm/tokens" > /softhsm/softhsm2.conf
```

Comment:
Ideally, this should have been part of the `meta-mender` PR, not the test.

Reply:
(but is okey! I only mention it for future reference)

Reply:
I do not think so -- I thought about it, and this is specific configuration for the package, it may some in handy but also may be disruptive if someone would like to do different tests. /softhsm/softhsm2.conf is specific to this test.